src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    public IsoscelesTrapezoid(String name, int a, int b, int h, String color) {
+        super(name, color);
+        setArea((a + b) / 2 * h);
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: " + this.getName()
+                + ", area: " + this.getArea() + " sq.units, "
+                + "color: " + this.getColor();
+    }
```

Comment:
lets replace method `toString()` to some our custom `drawFigure()` method in all classes

Reply:
I did it.